draft-js-mention-plugin/src/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Map } from 'immutable';
 import Mention from './Mention/Mention';
 import MentionSuggestions from './MentionSuggestions'; // eslint-disable-line import/no-named-as-default
-import MentionSuggestionsPortal from './MentionSuggestionsPortal';
+import MentionSuggestionsPortal from './MentionSuggestionsPortal/MentionSuggestionsPortal';
```

Comment:
I'm not sure why this nesting is need? I would flatten files.

Reply:
it makes sense

it was a lot of index.js, but I prefer to have the name in the file to make discovery easier on IDE

I or you can move files around